cli/src/project.ts

```diff
@@ -27,7 +27,8 @@ export default class Project {
   static async create(root: string, options: Partial<ProjectOptions> = {}): Promise<Project> {
     let { crate = 'native' } = options;
     const { stdout } = await execFile("cargo", ["metadata", "--format-version=1"], {
```

Comment:
Does adding `--no-deps` without increasing the buffer size resolve the issue?

The dependency isn't necessary, only the target path is used.

Reply:
Yes, just checked, it also helps

Reply:
I can modify PR to use --no-deps instead

Reply:
Please do. It should be faster and make the larger buffer unnecessary.

Reply:
done